### DISCUSSION TITLE
Anything for iPad?

### DISCUSSION THREAD

**Discussion by atatarn** [plantuml/plantuml#1338] [Q&A]
Hi there!

Don't you know, is there any tool to use PlantUML on iPad?
It would be great just to write diagram on iPad, save it to Dropbox and paste later into LaTeX-based documentation.

**Comment by mthhk** [comment]
+1

**Comment by deb-sandeep** [comment]
+1

**Comment by stefanfisk** [comment]
Since PlantUML is GPL it is not possible to distribute it via the App Store even if you manage to get it running on iOS.

**Comment by hgzapata** [comment]
According to this https://www.engadget.com/2011/01/09/the-gpl-the-app-store-and-you/  @stefanfisk is totally right

**Comment by hgzapata** [comment]
I found that there are distributions using MIT http://plantuml.com/download#mit   So perhaps it could be possible to have ports on iOS after all

**Comment by apollovy** [comment]
Hi folks. Sorry to resurrect this dinosaur, but I'm making a web app that is more user-friendly to mobile users. Now it has more neat view when on mobile and some other features. You can save this page as a shortcut on desktop and it would work just like an app. It requires internet connection though, because it uses plantuml.com API to render images.
More updates to come soon.
Everyone is welcome to make contributions.
http://apollov.tk/plantuml/
https://github.com/apollovy/plantuml-web

**Comment by BenceSzalai** [comment]
[PlantUML just retweeted](https://twitter.com/plantuml/status/1610998983425490945) few days ago that a new iPad editor has been released [in the AppStore](https://apps.apple.com/us/app/plantuml-app/id6444164984).
